### PR TITLE
Add Rollup to build node modules

### PR DESCRIPTION
**Pull request by Thomas-Geraghty**
- Added Rollup to build packages from the repo
- Added Rollup config for @ministryofjustice/hmpps-monitoring

### CODE REVIEW

File: packages/monitoring/package.json [hunk 1]
```diff
@@ -2,10 +2,14 @@
...
-  "types": "dist/index.d.ts",
+  "main": "./dist/index.cjs.js",
+  "module": "./dist/index.esm.js",
+  "types": "./dist/index.d.ts",
+  "files": [
+    "dist/**/*"
+  ],
   "scripts": {
-    "build": "tsc"
+    "build": "rollup -c rollup.config.js --bundleConfigAsCjs"
```
Comment (andrewrlee) [inline packages/monitoring/package.json:12]: does this do typechecking?

**Review by psoleckimoj — APPROVED** [review]
LGTM!

**Review by andrewrlee — APPROVED** [review]
LGTM

File: packages/monitoring/package.json (hunk 1 above)
Comment (andrewrlee) [reply packages/monitoring/package.json:12]: > noEmitOnError
Type: Boolean
Default: false
> If a type error is detected, the Rollup build is aborted when this option is set to true

[Do we need to enable this](https://www.npmjs.com/package/@rollup/plugin-typescript#:~:text=noEmitOnError,set%20to%20true.)?

Comment (Thomas-Geraghty) [reply packages/monitoring/package.json:12]: probably!

**Pull request merged**